src/content/docs/apm/agents/python-agent/troubleshooting/activate-application-warning-python.mdx

```diff
@@ -75,19 +78,20 @@ To solve this problem with your app:
 For other tips when no data appears, see the [Python troubleshooting documentation](/docs/agents/python-agent/troubleshooting/no-data-appears-python).
 
 ## Cause
-
-This is usually caused by a call to [`newrelic.agent.register_application`](/docs/agents/python-agent/python-agent-api/register_application) or [`newrelic.agent.application`](/docs/agents/python-agent/python-agent-api/application) occurring at import time.
-
-**Example**:
-
-```
-import newrelic.agent
-
-# This will cause the agent to activate whenever custom_event is imported
-app = newrelic.agent.application()
-
-def custom_event():
-    newrelic.agent.record_custom_event('CustomEvent', {}, application=app)
-```
-
-Code typically is loaded before a fork occurs in frameworks such as Celery or multiprocessing. As a result, the code is imported into the parent process instead of the child, launching the agent. Since the agent is launched in the parent process, no data is collected in the child process.
+There are two primary causes of this issue. The first is creating an application instance multiple times or creating an application instance prior to forking.
+
+1. This is usually caused by a call to [`newrelic.agent.register_application`](/docs/agents/python-agent/python-agent-api/register_application) or [`newrelic.agent.application`](/docs/agents/python-agent/python-agent-api/application) occurring at import time.
+
+    **Example**:
+    
+    ```
+    import newrelic.agent
+    
+    # This will cause the agent to activate whenever custom_event is imported
+    app = newrelic.agent.application()
+    
+    def custom_event():
+        newrelic.agent.record_custom_event('CustomEvent', {}, application=app)
+    ```
+
+1. This issue can also occur when monitoring services like celery where a main parent process launches worker processes. It occurs when an agent application instance is created on the main process prior to forking the worker processes. Since the agent is launched in the parent process, no data is collected in the child process.
```

Comment:
This number should be "2." instead of "1."  Or maybe we can use bullet points?  
< ul>
< li>first thing< /li>
< li>second thing< /li>
< /ul>

Reply:
Added number 2.